lib/turbostreamer/encoders/oj.rb

```diff
@@ -102,7 +107,7 @@ def capture(to=nil)
       result = output.string.gsub(/\A,|,\Z/, '')
       # Strip brackets as promised above
       if @stack.last == :map
-        result = result.gsub(/\A{\s*|\s*}\s*\Z/m, '') 
+        result = result.gsub(/\A{\s*|\s*}\s*\Z/m, '')
```

Comment:
Oh I wonder if these could be an issue too

Reply:
Maybe something like Ruby 2.5's `String#delete_prefix` and `String#delete_suffix` would be better here.

I know pre 2.5 there is `String#chomp`

Reply:
Changing `gsub` to `chomp` and `sub` doesn't cause performance improvement